mkdocs.yml

```diff
@@ -42,6 +42,12 @@ nav:
           - Vectors: R-IOC/r01_vectors.md
           - Operators: R-IOC/r02_operators.md
           - Functions: R-IOC/r03_functions.md
+          - Best Practices: R-IOC/r04_bestpractices.md
```

Comment:
I don't think the Best Practises section is properly placed. Let's think about a better position in the plan

Reply:
this part can be within the week 0 content as it's general instruction